src/main/java/com/mitsuki/jmatrix/util/Options.java

```diff
@@ -55,36 +72,37 @@ public enum OPT {
     private static String contentsPath = "contents/";
 
     /**
-    * Method that checks the input argument then returns specific option.
-    *
-    * @param  inputOpt  the {@code String} that wants to be checked.
-    *
-    * @return the corresponding {@code OPT} value.
-    *
-    * @since  1.0.0
-    * @see    Options#OPT
-    */
+     * Method that checks the input argument then returns specific option.
+     *
+     * @param  inputOpt                  the {@code String} that wants to be checked.
+     *
+     * @return                           the corresponding {@code OPT} value.
+     *
+     * @throws IllegalArgumentException  if the given option name does not
+     *                                   match with any known options.
+     *                                   This exception will be thrown as the causing exception.
+     *
+     * @since                            1.0.0b.1
+     */
     public static OPT getOptions(String inputOpt) {
         for (OPT opt : OPT.values()) {
             if (opt.aliases.contains(inputOpt)) {
                 return opt;
             }
         }
 
-        try {
-            throw new IllegalArgumentException(String.format("Unknown argument option for input \"%s\"", inputOpt));
-        } catch (final Exception e) {
-            try {
-                throw new JMBaseException(e);
-            } catch (final JMBaseException ex) {
-                raiseError(ex, 0);
-                System.err.println(System.lineSeparator() + getHelpMsg()[0]);
-                System.err.println("    " +
-                    "java -jar <jar_file> [-h|-V|-cr]");
-                System.exit(-1);
-            }
-        }
+        raiseError(new JMBaseException(
+            new IllegalArgumentException(
+                String.format("Unknown argument option for input \"%s\"", inputOpt)
+            )
+        ), 0);
+
+        System.err.println(System.lineSeparator() + getHelpMsg()[0]);
+        System.err.println("    " +
+            "java -jar <jar_file> [-h|-V|-cr]");
+        System.exit(-1);
```

Comment:
The `System.exit()` here is to makes program immediately stop and exit, and the `return` statement below it would never get executed and only to suppressed the `missing return statement` error.